product_secondary_unit/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_secondary_unit_user,access_product_second_unit_user,model_product_secondary_unit,base.group_user,1,0,0,0
+access_product_secondary_unit_user_stock,access_product_second_unit_user_stock,model_product_secondary_unit,stock.group_stock_user,1,1,1,0
+access_product_secondary_unit_manager,access_product_second_unit_manager,model_product_secondary_unit,stock.group_stock_manager,1,1,1,1
```

Comment:
You should either depend on `stock` or remove these permission lines.

I'm getting this error while trying to install the addon:

```
Exception: Module loading product_secondary_unit failed: file product_secondary_unit/security/ir.model.access.csv could not be processed:
 No matching record found for external id 'stock.group_stock_user' in field 'Group'
No matching record found for external id 'stock.group_stock_manager' in field 'Group'
```